test/libsolidity/syntaxTests/arrayLength/tuple_expression.sol

```diff
@@ -0,0 +1,11 @@
+contract C {
+    uint constant L1 = (2);
```

Comment:
Can you also add a more complex example with nested parenthesis?

Reply:
Done